tests/test_docker_latex.py

```diff
@@ -0,0 +1,51 @@
+import os
+import shutil
+import unittest
+import logging
+
+import quizgen.latex
+import quizgen.pdf
+import quizgen.util.dirent
+import quizgen.util.json
+import tests.base
+
+class TestDockerPdfConversion(tests.base.BaseTest):
+    def test_quiz_to_pdf(self):
+        if not quizgen.latex.is_available(use_docker = True):
+            self.skipTest("Docker is not installed or not running")
+
+        source_quiz_path = tests.base.discover_good_quiz_files()[0]
+
+        if not os.path.exists(source_quiz_path):
+            self.skipTest(f"Quiz file '{source_quiz_path}' not found")
+
+        temp_dir = quizgen.util.dirent.get_temp_path()
+        quiz_json_path = os.path.join(temp_dir, "quiz.json")
+
+        quizgen.util.dirent.copy_dirent(source_quiz_path, quiz_json_path)
+
+        original_base_dir = os.path.dirname(source_quiz_path)
+        quiz_data = quizgen.util.json.load_path(quiz_json_path)
+        
+        # Make all question paths absolute.
```

Comment:
This seems REALLY bad.

Why would you need to make all the question paths absolute?
This seems like a failure at some point in the process,
and you are covering it up with a band-aid ("plaster").
We want to address the core cause of issues, not cover them up.